R/tm_front_page.R

```
@@ -89,8 +100,8 @@ tm_front_page <- function(label = "Front page",
     server = srv_front_page,
     ui = ui_front_page,
     ui_args = args,
-    server_args = list(tables = tables, show_metadata = show_metadata),
-    datanames = if (show_metadata) "all" else NULL
+    server_args = list(tables = tables),
+    datanames = datanames
```

default datanames can't be NULL - `NULL` hides filter panel.